### ISSUE TITLE
Transfer sequences - not processing transcript sequences correctly

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/intermine#1181]
I think this is in the wrong order:

```
            } else if ("transfer-sequences".equals(operation)) {
                TransferSequences ts = new TransferSequences(getObjectStoreWriter());
                ts = new TransferSequences(getObjectStoreWriter());
                LOGGER.info("Starting TransferSequences.transferToLocatedSequenceFeatures()");
                ts.transferToLocatedSequenceFeatures();
                ts = new TransferSequences(getObjectStoreWriter());
                LOGGER.info("Starting TransferSequences.transferToTranscripts()");
                ts.transferToTranscripts();
```

`ts.transferToLocatedSequenceFeatures()` populates the sequence for all sequence features, `ts.transferToTranscripts()` only populates transcripts if they do not have a sequence. However this will never be the case because the transcripts would have gotten a sequence in 
`ts.transferToLocatedSequenceFeatures()`.

The order should be switched. @sergiocontrino Can you check my logic?

**Comment by julie-sullivan** [comment]
Update the unit test to include introns. This will test which method is actually updating the sequence.

**Comment by sergiocontrino** [comment]
tested that the proposed change does not interfere with current process. logic seems fine as well.

**Comment by julie-sullivan** [comment]
Solution: set the transcript method to not do a NULL check. This will set all transcripts in the database regardless of source. This may or may not be desirable. Will need some more thought.

**Comment by julie-sullivan** [comment]
@sergiocontrino can you confirm we can close this ticket? I believe this issue was solved by you in 00f914e2f977a8765d2a50275dbd554b3aa73c2b

**Comment by sergiocontrino** [comment]
yes, i think this is solved. closing.
